docs/kata-containers.md

```diff
@@ -0,0 +1,44 @@
+# Kata containers support in CRI-RM
```

Comment:
i'd suggest to use a shorter title like `Kata Containers`. Especially if this is going to be a separate title on the main level. The headings should be in Capital Case